fiftyone/operators/store/service.py

```
@@ -1,43 +1,28 @@
-
-    Args:
-        dataset_id: the dataset ID
-    """
-    svc = ExecutionStoreService(dataset_id=dataset_id)
-    svc.cleanup_for_dataset()
-
-
 class ExecutionStoreService(object):
     def __init__(
         self,
         repo: Optional["ExecutionStoreRepo"] = None,
```

Ruff

Undefined name `ExecutionStoreRepo`

(F821)